src/web/app/src/components/SignUp/Forms/RSSFeeds.tsx

```diff
@@ -77,7 +77,7 @@ const useStyles = makeStyles((theme: Theme) =>
       width: '70%',
       padding: '4px',
       alignSelf: 'center',
-      fontSize: '0.8em',
+      fontSize: '1.1em',
```

Comment:
## Optimal font size
There are no exact rules for font sizing, but there are some generally good practices to have Font sizes around `1.6rem` to `1.8rem`.

I would really appreciate to see other views as well on this. 
@humphd 
Thanks

Reply:
Let's leave this for a follow-up, if you want to file an issue on it.